src/quacc/atoms/skzcam.py

```diff
@@ -23,13 +26,701 @@
+            if "frozencore" not in oniom_parameters:
+                raise ValueError(
+                    "The frozencore must be provided for all ONIOM levels."
+                )
+            if oniom_parameters["max_cluster_num"] > len(
+                self.quantum_cluster_indices_set
+            ):
+                raise ValueError(
+                    "The maximum cluster number for all ONIOM levels must be less than or equal to the number of quantum clusters."
+                )
+
+    def create_element_info(
+        self,
+        frozencore: Literal["valence", "semicore"] | dict[ElementStr, int],
+        basis: Literal["DZ", "TZ", "QZ", "5Z"] | dict[ElementStr, str],
+        code: Literal["mrcc", "orca"],
+        ecp: dict[ElementStr, str],
+        ri_scf_basis: dict[ElementStr, str] | None = None,
+        ri_cwft_basis: dict[ElementStr, str] | None = None,
+    ):
+        """
+        Creates the element info dictionary for the SKZCAM input across each oniom layer.
+
+        Parameters
+        ----------
+        basis
+            The basis set to use for the quantum cluster. This could be either double-zeta, triple-zeta, quadruple-zeta, quintuple-zeta, denoted as 'DZ', 'TZ',' QZ' and '5Z' respectively.
+        frozencore
+            The frozen core to use for the quantum cluster. This could be specified as a string being either 'semicore' or 'valence'
+        code
+            The code to use for the quantum cluster. This could be either mrcc or orca.
+        ecp
+            The effective core potential to use for each element within the quantum cluster.
+
+        Returns
+        -------
+        dict[ElementStr, ElementInfo]
+            A dictionary with elements as keys which gives the (1) number of core electrons as 'core', (2) basis set as 'basis', (3) effective core potential as 'ecp', (4) resolution-of-identity/density-fitting auxiliary basis set for DFT/HF calculations as 'ri_scf_basis' and (5) resolution-of-identity/density-fitting for correlated wave-function methods as 'ri_cwft_basis'.
+        """
+
+        # Create an adsorbate_slab_quantum_cluster object for the first SKZCAM cluster
+        adsorbate_slab_quantum_cluster = self.adsorbate_slab_embedded_cluster[
+            self.quantum_cluster_indices_set[0]
+        ]
+
+        # Define set of elements
+        set_of_elements = list(
+            set(adsorbate_slab_quantum_cluster.get_chemical_symbols())
+        )
+
+        # Choose whether to provide preset values for the element info dictionary based on the SKZCAM paper:
+        # Preset values are used if basis is not a dictionary or frozencore is not a dictionary
+        use_presets = False
+        if (
+            isinstance(basis, str)
+            and isinstance(frozencore, str)
+            and basis in ["DZ", "TZ", "QZ", "5Z"]
+            and frozencore in ["valence", "semicore"]
+        ):
+            use_presets = True
```

Comment:
Do the `if` and then `else: use_presets = False` rather than hard-coding it above.